### PR TITLE
GEODE-6745: Add status to list members output for REST v2

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
Authored-by: Jens Deppe <jdeppe@pivotal.io>

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jdeppe-pivotal** [comment]
@Petahhh

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetMemberInformationFunction.java [hunk 1]
```diff
@@ -142,12 +139,22 @@ public void execute(FunctionContext functionContext) {
           numConnections = numConnections + status.getNumberOfConnections();
         }
         memberInfo.setClientCount(numConnections);
+
+        ServerLauncher.ServerState state = ServerLauncher.getServerState();
+        if (state != null) {
```
Comment (Petahhh) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetMemberInformationFunction.java]: Are we not handling the null case? When the state is null, should we throw an error? Should we provide a default status?

@jdeppe-pivotal

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetMemberInformationFunction.java [hunk 2]
```diff
@@ -142,12 +139,22 @@ public void execute(FunctionContext functionContext) {
...
+          memberInfo.setStatus(state.getStatus().getDescription());
+        }
       } else {
         memberInfo.setServer(false);
         InternalLocator locator = InternalLocator.getLocator();
         if (locator != null) {
           memberInfo.setLocatorPort(locator.getPort());
         }
+
+        LocatorLauncher.LocatorState state = LocatorLauncher.getLocatorState();
```
Comment (Petahhh) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetMemberInformationFunction.java]: This block looks like duplicate code.

Can we create a empty final variable called `state` which is assigned within the if/else. Then outside the if/else we can `memberInfo.setStatus(state.getStatus().getDescription());`

Comment (Petahhh) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetMemberInformationFunction.java]: @jdeppe-pivotal

**Pull request merged**